### ISSUE TITLE
Autolayout state machine on connect.

### ISSUE THREAD

**Issue by TheDudeFromCI** [TheDudeFromCI/mineflayer-statemachine#5]
The default state machine layout, a simple circle, is very poor and requires the user to manually reorganize the scene completely after each reloads. This can be annoying.

If the nodes were laid out automatically on connect, this process would be much easier and would require almost no action on the user's part.

**Comment by sefirosweb** [comment]
I'm agreed!

I'm making a complex bot, using your pluging bot state,

Every time when i recharger I must have reorganize tthe diagram =P

Maybe the "fast" solution is set X and Y position?

Similar:
```js
  const checkItemsToDeposit = new BehaviorIdle(targets)
  checkItemsToDeposit.stateName = 'Verify Items To Deposit'
  checkItemsToDeposit.x = 100
  checkItemsToDeposit.y = 100
```

Of course this order is manually, but is better than nothing

![image](https://user-images.githubusercontent.com/20754836/98439136-08fd7400-20f0-11eb-94de-d2db4195b244.png)


Ordered:
![image](https://user-images.githubusercontent.com/20754836/98439127-f5520d80-20ef-11eb-8b18-00d96f694c6d.png)

**I know i can use nested for optimize the code

**Comment by sefirosweb** [comment]
Hi the dude! I did i myself :D,

This as your example of web server:
![image](https://user-images.githubusercontent.com/20754836/98443317-d0b75f00-210a-11eb-8739-5020c40f397f.png)

If the user don't puts a different X or Y they run your "round" diagram
![image](https://user-images.githubusercontent.com/20754836/98443370-255ada00-210b-11eb-8440-12c7b8ec8016.png)

But i runned npx standard --fix for standardized the code and i see alot mismatch with your code,

How I can standardize the code with you?

![image](https://user-images.githubusercontent.com/20754836/98443299-bda48f00-210a-11eb-8153-f443faaeae0b.png)

Edit:
Oh I see in your topics is not installed standard, for that i have a lot mismatching,
I make a pull request, and you can take a look?

Best x2!

**Comment by TheDudeFromCI** [comment]
@sefirosweb Okay, I finished the PR for installing Standard. #74 
Once that is merged, you should be all good with your PR.

**Comment by sefirosweb** [comment]
Hi @TheDudeFromCI  i made a new pull request :D wihouth alot of changes,

For huges workflos & statemachines this can be help us

Best!

**Comment by TheDudeFromCI** [comment]
Merged
